recipes/radio_beam/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "radio_beam" %}
+{% set version = "0.2" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "3b3b3cfcc17d61df469c9e45d7f7fc993b902f3f2c076279071fd984a6e5763d" %}
+
+package:
+  name: radio-beam
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - pip
+    - astropy
+    - numpy >=1.9.0
+  run:
+    - python
+    - astropy
+    - numpy >=1.9.0
+
+test:
+  imports:
+    - radio_beam
+
+about:
+  home: https://github.com/radio-astro-tools/radio-beam
+  license: BSD 3-clause
```

Comment:
I think the correct license qualifier is `BSD-3-Clause`, as in [here](https://opensource.org/licenses/alphabetical)